packages/manager/modules/pci/src/projects/project/private-registry/list/index.js

```diff
@@ -0,0 +1,59 @@
+import angular from 'angular';
+import '@uirouter/angularjs';
+import listComponent from './list.component';
+
+const moduleName = 'ovhManagerPciProjectPrivateRegistryList';
+
+angular
+  .module(moduleName, [
+    'ui.router',
+  ])
+  .config(/* @ngInject */($stateProvider) => {
+    $stateProvider.state('pci.projects.project.private-registry.list', {
+      url: '/list',
+      component: 'listComponent',
+      translations: {
+        value: [
+          './..',
+          '.',
+        ],
+        format: 'json',
+      },
+      resolve: {
+        breadcrumb: () => null, // Hide breadcrumb
+      },
+    })
+      .state('pci.projects.project.private-registry.list.create', {
```

Comment:
Avoid to create multiple state in same file (and define it in `*.routing.js`).
If you have the same state in 2 different places (like creation from `list` or creation from `onboarding`) you should declare a component and have 2 sub-states that uses this component (like `users` : https://github.com/ovh-ux/manager/blob/master/packages/manager/modules/pci/src/projects/project/users/onboarding/add/add.module.js#L4)